demisto_sdk/commands/doc_reviewer/tests/conftest.py

```diff
@@ -59,3 +65,162 @@ def misspelled_integration(invalid_spelled_content_pack):
     Returns a misspelled integration.
     """
     return invalid_spelled_content_pack[0].integrations[0]
+
+
+@pytest.fixture(name="supported_pack")
+def valid_spelled_xsoar_supported_content_pack(tmp_path: PosixPath) -> Pack:
+    """
+    Create a Pack with valid spelled content.
+    """
+
+    temp_pack_dir = tmp_path / "Packs"
+    temp_pack_dir.mkdir()
+
+    temp_repo = SimpleNamespace()
+    setattr(temp_repo, "path", str(temp_pack_dir / ".git"))
+
+    pack = Pack(packs_dir=temp_pack_dir, name="TempPack", repo=temp_repo)
+    pack.create_release_notes(
+        version="release-note-1",
+        content="\n#### Scripts\n##### ScriptName\n- Added the feature.",
+    )
+    pack.pack_metadata.update({"support": "xsoar"})
```

Comment:
Can we use here the constants we used in the code? (for Support, community, and xsoar)?